databases/week2/hw2-1.sql

```diff
@@ -0,0 +1,70 @@
+--Part1
+use hyf;
+--ADD new task
+INSERT INTO task (id,title,description,created,updated,due_date,status_id,user_id)
+VALUES (16,'Finish Homework', 'Complete all SQL tasks for the week', NOW(), NOW(), '2025-02-15', 1, 2);
```

Comment:
You are using the schema from week 1. You should be using this data: [Link](https://github.com/HackYourFuture-CPH/databases/blob/main/lesson2/lesson2-data.sql)